spec/support/requests_shared_examples.rb

```diff
@@ -213,6 +213,32 @@
   end
 end
 
+shared_examples 'a sorted index resource' do |expected_last_item_sym|
+  let(:expected_last_item) { send(expected_last_item_sym) }
+  let(:sort_column) { :created_at }
+  let(:sort_order) { "desc" }
+
+  it 'should return a sorted index' do
+    is_expected.to eq(expected_response_status)
+    response_json = JSON.parse(response.body)
+    expect(response_json).to have_key('results')
+    returned_results = response_json['results']
+    expect(returned_results).to be_a(Array)
```

Comment:
It would be good to test that `returned_results` has more than one element.

Reply:
done

Reply:
Almost. I'll rephrase. Ensure `returned_results` has two or more elements. It's not useful to test that an array with a single element is sorted properly...